man/systemd.unit.xml

```diff
@@ -1023,6 +1023,19 @@
         <xi:include href="version-info.xml" xpointer="v201"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>SurviveFinalKillSignal=</varname></term>
+
+        <listitem><para>Takes a boolean argument. Defaults to <option>no</option>. If <option>yes</option>,
+        processes belonging to this unit will not be sent the final <literal>SIGTERM</literal> and
+        <literal>SIGKILL</literal> on state transition (e.g.: shutdown).
```

Comment:
on state → signals on state

Reply:
"state transition"? that's a bit too generic. Say "during the final phase of the system shutdown process"?